lang/cs/Org.Apache.REEF.Client/AzureBatch/Service/AzureBatchService.cs

```diff
@@ -39,21 +40,39 @@ public sealed class AzureBatchService : IDisposable
         public string PoolId { get; private set; }
 
         private BatchClient Client { get; set; }
+        private ContainerRegistryProvider ContainerRegistryProvider { get; set; }
+        private IList<string> Ports { get; set; }
+        private ICommandBuilder CommandBuilder { get; set; }
```

Comment:
here and in all properties above: can remove `set` accessors

Reply:
Got it. Will take care of it. Thanks!

---
In reply to: [203899897](https://github.com/apache/reef/pull/1478#discussion_r203899897) [](ancestors = 203899897)